src/app/config.rs

```diff
@@ -371,9 +386,16 @@ impl Config {
         let admin_listener_addr = parse(strings, ENV_ADMIN_LISTEN_ADDR, parse_socket_addr);
         let inbound_forward = parse(strings, ENV_INBOUND_FORWARD, parse_socket_addr);
 
+        let inbound_dispatch_timeout = parse(strings, ENV_INBOUND_DISPATCH_TIMEOUT, parse_duration);
```

Comment:
Nitpicky: Should we have `parse_timeout` function for these that takes INBOUND_BASE/OUTBOUND_BASE/CONTROL_BASE and a "kind" parameter (connect/dispatch)? It seems like other stuff that's configurable separately for inbound/outbound/ctrl is moving in that direction...

Reply:
Sure, but let's make that it's own change if/when we do it.